components/ide/jetbrains/backend-plugin/BUILD.yaml

```diff
@@ -151,6 +151,7 @@ packages:
       - NO_VERIFY_JB_PLUGIN=true
     config:
       commands:
+        - ["rm", "-rf", "src/main/kotlin/io/gitpod/jetbrains/remote/listeners/GradleSyncListener.kt"]
```

Comment:
We need this line because the file `exclude` in `build.gradle.kt` somehow not working well. (And I don't want to investigate why)